code/modules/research/designs/medical_designs.dm

```diff
@@ -242,6 +242,17 @@
 	build_path = /obj/item/organ/internal/cyberimp/chest/thrusters
 	category = list("Misc", "Medical Designs")
 
+/datum/design/toolarm
+	name = "Toolbox Arm"
+	desc = "A stripped-down version of engineering cyborg toolset, designed to be installed on subject's arm."
+	id = "tool_arm"
+	req_tech = list("materials" = 4, "engineering" = 3, "biotech" = 4, "powerstorage" = 4)
+	build_type = PROTOLATHE | MECHFAB
+	materials = list (MAT_METAL = 2000, MAT_GLASS = 750, MAT_SILVER = 600)
```

Comment:
Reminder a single sheet of material is 2k units.

Probably wont matter either way since people are drowning in every material but diamond and uranium, but being able to make ~12 of these on average from mining a single silver vein is silly. It's basically free.